specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2022-10-31-preview/iotcentral.json

```diff
@@ -0,0 +1,9326 @@
+        }
+      },
+      "required": [
+        "queryRange"
+      ]
+    },
+    "BlobStorageV1Destination": {
+      "description": "The blob storage destination definition.",
+      "x-ms-discriminator-value": "blobstorage@v1",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Destination"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "authorization": {
+          "$ref": "#/definitions/BlobStorageV1DestinationAuth"
+        }
+      },
+      "required": [
+        "authorization"
+      ]
+    },
+    "BlobStorageV1DestinationAuth": {
+      "description": "The authentication definition for blob storage destination.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "The kind of authentication to use.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "type"
+      ],
+      "discriminator": "type"
+    },
+    "BlobStorageV1DestinationConnectionStringAuth": {
+      "description": "The authentication definition with connection string for blob storage destination.",
+      "x-ms-discriminator-value": "connectionString",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BlobStorageV1DestinationAuth"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "connectionString": {
+          "description": "The connection string for accessing the blob storage account.",
+          "type": "string"
+        },
+        "containerName": {
+          "description": "Name of the container where data should be written in the storage account.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "connectionString",
+        "containerName"
+      ]
+    },
+    "BlobStorageV1DestinationSystemAssignedManagedIdentityAuth": {
+      "description": "The authentication definition with system assigned managed identity for blob storage destination.",
+      "x-ms-discriminator-value": "systemAssignedManagedIdentity",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BlobStorageV1DestinationAuth"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "endpointUri": {
+          "description": "The storage account's blob service endpoint URL.",
+          "type": "string"
+        },
+        "containerName": {
+          "description": "Name of the container where data should be written in the storage account.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "endpointUri",
+        "containerName"
+      ]
+    },
+    "CapabilityAggregateFunctionType": {
+      "description": "The type of aggregation to be applied on capability data.",
+      "type": "string",
+      "default": "count",
+      "enum": [
+        "sum",
+        "count",
+        "max",
+        "min",
+        "avg"
+      ],
+      "x-ms-enum": {
+        "name": "CapabilityAggregateType",
+        "modelAsString": false,
```

Comment:
With `false`, you'll never be able to add functions. Are you 100% sure? Otherwise, use `true`.

Reply:
Hi @heaths,  this should be "true" as extensible enumeration in future, same as deviceType, destinationSource, ImageTileTextUnits, I have fixed from the new swagger file.